cli/cmd/lql.go

```diff
@@ -19,71 +19,217 @@
 package cmd
 
 import (
+	"fmt"
+	"io"
 	"io/ioutil"
+	"net/http"
+	"strings"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
 )
 
 var (
-	lqlFile string
+	lqlEnd      string
+	lqlEnv      bool
+	lqlFile     string
+	lqlRepo     bool
+	lqlStart    string
+	lqlURL      string
+	lqlValidate bool
 
-	// lqlCmd represents the lql command
+	// lqlCmd represents the lql parent command
 	lqlCmd = &cobra.Command{
-		Use:    "lql <query>",
-		Hidden: true,
-		Short:  "run an LQL query",
+		Aliases: []string{"lql"},
+		Use:     "query",
+		Short:   "Run and manage LQL queries",
```

Comment:
short descriptions start with lowercase
```suggestion
		Short:   "run and manage LQL queries",
```

Reply:
Fixed.